jenkins-pipelines/longevity-ics-nemesis-alter-table-1tb-7d.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
```

Comment:
please change the file name to:
longevity-ics-toggle-strategy-500gb-24h.jenkinsfile

Reply:
done